packages/graphql/src/services/courses.ts

```diff
@@ -523,10 +523,12 @@ export async function getCourseData(
           updatedAt: 'desc',
         },
       },
-      microSessions: {
```

Comment:
Is this change really required? Currently, we only show the number of questions and for the new UI I thought it would make sense to show the "number of question sets" = stacks